docs/.pages

```diff
@@ -9,6 +9,7 @@ nav:
   - Reference: reference
   - Architecture: architecture
   - Troubleshooting: troubleshooting
+  - Known Issues: known-issues
```

Comment:
Should we name it known limitations?

Reply:
Sorry, that was just a question - I'm okay with known issues as well. @edipascale I you feel better about known issues name - please do another PR just to rename it back

cc @pau-hedgehog @mrbojangles3